ansible_base/rbac/api/views.py

```diff
@@ -160,7 +160,9 @@ class RoleTeamAssignmentViewSet(BaseAssignmentViewSet):
 
     serializer_class = RoleTeamAssignmentSerializer
     prefetch_related = ('team',)
-    filter_backends = (TeamAnsibleIdAliasFilterBackend,)
+    filter_backends = BaseAssignmentViewSet.filter_backends + [
```

Comment:
this feels weird but I don't know how else to extend to the list
i.e. we want to preserve the `DEFAULT_FILTER_BACKENDS` list but append just one filter for this viewset